db/schema_tables.cc

```diff
@@ -1736,6 +1742,14 @@ mutation make_scylla_tables_mutation(schema_ptr table, api::timestamp_type times
     // In-memory tables are deprecated since scylla-2024.1.0
     // FIXME: delete the column when there's no live version supporting it anymore.
     // Writing it here breaks upgrade rollback to versions that do not support the in_memory schema_feature
+    if (auto tablet_hints_cdef = scylla_tables()->get_column_definition("tablet_hints")) {
```

Comment:
Hmm, this if is always true because scylla_tablets() works with a full feature set.

Reply:
Gone in v12 ([f8a525c](https://github.com/scylladb/scylladb/commit/f8a525ce4d97fdaa7221b9f94ec08546ebe59112))